src/deep-core/lib/AWS/Lambda/Runtime.js

```diff
@@ -150,6 +152,68 @@ export class Runtime extends Interface {
   }
 
   /**
+   * @returns {null|Object}
+   */
+  get calleeConfig() {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (11)](https://www.codacy.com/app/MitocGroup/deep-framework/file/1885412730/issues/source?bid=2895475&fileBranchId=3089240#l157)